README.md

```diff
@@ -84,6 +84,11 @@ var config = new Config
   };
 var client = new Client(config);
 ~~~~
+To parse the terminal API notifications, please use the following custom deserializer:
+~~~~ csharp
+var serializer = new SaleToPoiMessageSerializer();
+var saleToPoiRequest = serializer.DeserializeNotification(your_terminal_notification);
```

Comment:
The DeserializeNotification can throw an Exception. See line 97 above. It's good to let the developer that use this know.

```csharp
 throw new Exception("Json input is not a terminal notification.");
```

[Consider] this Deserialize function could be a static utility class*